src/main/java/db/Storage.java

```diff
@@ -0,0 +1,11 @@
+package db;
+
+import java.util.Optional;
+
+public interface Storage {
+    void saveToStorage(String product, int quantity);
```

Comment:
Standard style is to leave one empty line after class/interface declaration and one line after the end of class/interface

Reply:
Thank you, done!